frontend/src/components/TaskView/FutureView/FutureViewDayTaskContainer.tsx

```diff
@@ -54,23 +59,57 @@ function FutureViewDayTaskContainer(
     setPrevHeights([tasksHeight, containerHeight]);
     onHeightChange(tasksHeight > containerHeight && containerHeight > 0, tasksHeight);
   });
+  const [localTasks, setLocalTasks] = useState<IdOrder[]>(idOrderList);
+  const onDragEnd = (result: DropResult): void => {
+    const { source, destination } = result;
+    if (destination == null) {
+      // invalid drop, skip
+      return;
+    }
+    const sourceOrder: number = source.index;
+    const dest = localTasks[destination.index];
+    const destinationOrder: number = dest == null ? sourceOrder : dest.order;
 
-  const taskListComponent = idOrderList.map(({ id }) => (
-    <FutureViewTask
-      key={id}
-      taskId={id}
-      containerDate={date}
-      inNDaysView={inNDaysView}
-      taskEditorPosition={taskEditorPosition}
-      calendarPosition={calendarPosition}
-      doesShowCompletedTasks={doesShowCompletedTasks}
-      isInMainList={isInMainList}
-    />
+    const reorderMap = computeReorderMap(localTasks, sourceOrder, destinationOrder);
+    setLocalTasks(getReorderedList(localTasks, reorderMap));
+    applyReorder('tasks', reorderMap);
+  };
+  const taskListComponent = idOrderList.map(({ id, order }) => (
+    <Draggable key={id} draggableId={id} index={order}>
+      { (provided) => (
+        <div ref={provided.innerRef} {...provided.draggableProps} {...provided.dragHandleProps}>
+          <FutureViewTask
+            key={id}
+            taskId={id}
+            containerDate={date}
+            inNDaysView={inNDaysView}
+            taskEditorPosition={taskEditorPosition}
+            calendarPosition={calendarPosition}
+            doesShowCompletedTasks={doesShowCompletedTasks}
+            isInMainList={isInMainList}
+          />
+        </div>
+      )}
+    </Draggable>
   ));
   if (isInMainList) {
-    const style = { overflow: 'hidden' };
+    const style = {};
     return (
-      <div className={styles.Container} style={style} ref={containerRef}>{taskListComponent}</div>
+      <DragDropContext onDragEnd={onDragEnd}>
+        <Droppable droppableId="onetestyboy">
```

Comment:
It's OK to have fun in the code, but you should pick some name that is both
- fun
- at least reveal some information about the object you are modeling.
Here is a good example in my opinion: https://github.com/cornell-dti/samwise/blob/37eca33929116f626b806c08d1da266384c23811/frontend/src/util/tag-util.ts#L6
`onetestyboy` here simply adds confusion.

Reply:
![image](https://user-images.githubusercontent.com/7517829/69502156-e91e7180-0ed9-11ea-9856-2d9c78eb11a9.png)
does this fit the criteria? 🤔

Reply:
Droppable ID has been changed to the boring "future-view-task-droppable"